addons/mail/static/tests/qunit_suite_tests/components/link_preview_tests.js

```diff
@@ -0,0 +1,415 @@
+    pyEnv['mail.message'].create({
+        body: 'not empty',
+        link_preview_ids: [linkPreviewId],
+        message_type: 'comment',
+        model: "mail.channel",
+        res_id: mailChannelId,
+    });
+    const { openDiscuss } = await start({
+        discuss: {
+            params: {
+                default_active_id: `mail.channel_${mailChannelId}`,
+            },
+        },
+    });
+    await openDiscuss();
+
+    assert.containsOnce(
+        document.body,
+        '.o_LinkPreviewVideoView',
+        "should have link preview video in DOM"
+    );
+    assert.containsOnce(
+        document.body,
+        '.o_LinkPreviewVideoView_title',
+        "Should display the link preview title"
+    );
+    assert.containsOnce(
+        document.body,
+        '.o_LinkPreviewVideoView_description',
+        "Link preview should show the link description"
+    );
+    assert.containsOnce(
+        document.body,
+        '.o_linkPreviewVideo_overlay',
+        "Should display overlay inside the link preview video image"
+    );
+});
+
+QUnit.test('Link preview image layout', async function (assert) {
+    assert.expect(1);
+
+    const pyEnv = await startServer();
+    const linkPreviewId = pyEnv['mail.link.preview'].create(link_preview_image_payload);
+    const mailChannelId = pyEnv['mail.channel'].create({});
+    pyEnv['mail.message'].create({
+        body: 'not empty',
+        link_preview_ids: [linkPreviewId],
+        message_type: 'comment',
+        model: "mail.channel",
+        res_id: mailChannelId,
+    });
+    const { openDiscuss } = await start({
+        discuss: {
+            params: {
+                default_active_id: `mail.channel_${mailChannelId}`,
+            },
+        },
+    });
+    await openDiscuss();
+
+    assert.containsOnce(
+        document.body,
+        '.o_LinkPreviewImageView',
+        "should have link preview image"
+    );
+});
+
+QUnit.test('Remove link preview Gif', async function (assert) {
+    assert.expect(1);
+
+    const pyEnv = await startServer();
+    const linkPreviewId = pyEnv['mail.link.preview'].create(link_preview_gif_payload);
+    const mailChannelId = pyEnv['mail.channel'].create({});
+    pyEnv['mail.message'].create({
+        body: 'not empty',
+        link_preview_ids: [linkPreviewId],
+        message_type: 'comment',
+        model: "mail.channel",
+        res_id: mailChannelId,
+    });
+    const { openDiscuss, click, mouseenter } = await start({
+        discuss: {
+            params: {
+                default_active_id: `mail.channel_${mailChannelId}`,
+            },
+        },
+    });
+    await openDiscuss();
+
+    await mouseenter('.o_LinkPreviewImageView');
+    await click('.o_LinkPreviewAside');
+    assert.containsOnce(
+        document.body,
+        '.o_LinkPreviewDeleteConfirm',
+        'Should have a link preview confirmation dialog'
+    );
+});
+
+QUnit.test('Remove link preview card image', async function (assert) {
```

Comment:
"card image" is confusing written this way as they are 2 different types of preview

Reply:
I removed this test since it's the same as regular card.